waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -309,3 +311,12 @@ getPackageJsonOverrides = map buildOverrideData (designateLastElement overrides)
     designateLastElement l =
       map (\(x1, x2, x3) -> (x1, x2, x3, False)) (init l)
         ++ map (\(x1, x2, x3) -> (x1, x2, x3, True)) [last l]
+
+genValidators :: Generator [FileDraft]
+genValidators =
+  return
+    [ genSharedTmplToServerSrcDirCopy [relfile|validators.js|] [relfile|validators/validators.js|]
+    ]
+
+genSharedTmplToServerSrcDirCopy :: Path' (Rel S.SharedTemplatesDir) File' -> Path' (Rel C.ServerSrcDir) File' -> FileDraft
+genSharedTmplToServerSrcDirCopy src = S.mkTmplFdWithDst src . (</>) C.serverSrcDirInProjectRootDir
```

Comment:
**Nit:** I think the pointfree style makes things less readable in this case, so I'd probably specify both arguments explicitly (we'd get to name the other argument too).

If you decide to stick with the pointfree approach, I recommend changing `(</>) C.serverSrcDirInProjectRootDir` to `(</> C.serverSrcDirInProjectRootDir)`.